docs/machine-learning/resources/azure-training-concepts-model-builder.md

```diff
@@ -59,7 +59,23 @@ Once training is complete, two projects are added to your solution with the foll
   - bestModel.onnx: A serialized version of the model in Open Neural Network Exchange (ONNX) format. ONNX is an open source format for AI models that supports interoperability between frameworks like ML.NET, PyTorch and TensorFlow.
   - bestModelMap.json: A list of categories used when making predictions to map the model output to a text category.
   - MLModel.zip: A serialized version of the ML.NET prediction pipeline that uses the serialized version of the model *bestModel.onnx* to make predictions and maps outputs using the `bestModelMap.json` file.
-  
+
+## Consume the model
```

Comment:
Are we set on using the word "Consume" @briacht?

I did some SEO keyword research and "Consume machine learning model" has zero search volume (the average number of times per month that a keyword was queried in a search engine), whereas "Use machine learning model" has a volume of 70, which is still lowish but not zero

Reply:
I think Use machine learning model sounds simpler / better.

Reply:
resolved